db.tf

```diff
@@ -102,7 +102,12 @@ resource "null_resource" "db_iam_auth" {
   depends_on = ["aws_rds_cluster_instance.apiary_cluster_instance"]
 
   provisioner "local-exec" {
-    command = "${path.module}/scripts/db-iam-auth.sh ${aws_rds_cluster.apiary_cluster.endpoint} ${aws_rds_cluster.apiary_cluster.master_username} '${aws_rds_cluster.apiary_cluster.master_password}'"
+    command = "${path.module}/scripts/db-iam-auth.sh"
+    environment {
+        MYSQL_HOST="${aws_rds_cluster.apiary_cluster.endpoint}"
+        MASTER_USER="${aws_rds_cluster.apiary_cluster.master_username}"
```

Comment:
Can we prefix this and the next one with MYSQL_ too? That way all MySql related environment variables are consistently named.